minetest/minetest
# Increase the resolution of the logo shown on the about tab

The Minetest logo on the about tab only has a size of 100x100 pixels, so it is scaled up and looks pixelated. This PR increases its resolution to 200x200 pixels.

Before this PR (first Android, then desktop):
![screenshot before 1](https://github.com/minetest/minetest/assets/82708541/7373b7d8-fa16-4856-a0ee-5098cb761592) ![screenshot before 2](https://github.com/minetest/minetest/assets/82708541/f02493f9-7314-47eb-b2c6-dcc734536e0f) 

After this PR (first Android, then desktop):
![screenshot after 1](https://github.com/minetest/minetest/assets/82708541/2aa18c99-6174-4c98-b563-cb46d5aefaca) ![screenshot after 2](https://github.com/minetest/minetest/assets/82708541/81a2127c-8ac9-4c1c-84b2-0674db6cb62f) 

The logo still looks quite bad because Minetest doesn't seem to do proper interpolation when scaling, but I think it's an improvement. The file changed by this PR (`textures/base/pack/logo.png`) is also used as the window icon. This change doesn't cause any problems with the window icon for me (Linux, X11).

## To do

This PR is a Ready for Review.

## How to test

Open the about tab, verify that the Minetest logo looks less bad than before. Also verify that the window icon is still set correctly.
